packages/scripts/src/analyze-feature.ts

```diff
@@ -171,6 +171,8 @@ export function loadFeaturesFromPaths(
                 contextFilePaths: {},
                 resolvedContexts: {},
                 isRoot: ownFeatureFilePaths.has(featureFilePath),
+                packageName: featurePackage.name,
```

Comment:
I need to collect these too, so that I could use them when mapping external features for webpack